docs/guidelines.md

```diff
@@ -0,0 +1,87 @@
+high level of fluency at a low level of proficiency. We're aiming for fluency
+in the syntax, idioms, and the standard library of the language. You can read
+more about the [goal of exercism here](https://github.com/exercism/docs/blob/master/about/goal-of-exercism.md).
+
+## Definitions
+
+In the following paragraphs, keywords such as **MUST**, **SHOULD**, **COULD**
+are to be interpreted as in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt);
+given that we recognise the following four output states and their restrictions:
+
+- `approve_as_optimal`: **COULD** be a best solution, **SHOULD** be without comment
+- `approve_with_comment`: **COULD** be a best solution, **MUST** be with comment
+- `disapprove_with_comment`: **MUST** be with comment
+- `refer_to_mentor`: **COULD** be with comment
+
+## Conditions and outcome
+
+For feedback generated by automated mentoring:
+
+- it **SHOULD** `disapprove_with_comment` when someone isn't using one of the 
+  subjects the exercise is supposed to teach,
+- each `disapprove_with_comment` **MUST** has at least one comment,
+- each `disapprove_with_comment` **SHOULD** steer a student towards an `approve_`,
+- a student **SHOULD NOT** get a `refer_to_mentor` in a newer iteration, if they've
+  had a `disapprove_with_comment` in a previous iteration,
+- a `refer_to_mentor` **COULD** have one or more comments which will be given to
+  the mentor.
+
+There is no minumum or maximum amount of :speech_balloon: comments. However, each
+comment **SHOULD** be aiding the student towards an approvable solution.
+
+## Approvability
+
+Given the above, and to re-iterate that we focus on language fluency:
+
+### Incorrect Naming or Casing
+
+In general, if it's a _language feature_ that will be caught by a compiler or parser
+or based on official rules (which means there is a dependency on correctness in 
+tools), you should `disapprove_with_comment`, preferably linking to the official
+rules. If it's a _stylistic_ choise, or there are _competing standards_, do not
+remark at all.
+
+> - :-1: dissapprove if there are official *strict* guidelines on naming conventions
+> - :+1: approve if there are only _defacto_ standard(s) and everything else is fine.
+> - :speech_balloon: leave a comment if there is something noteworthy
+> - :no_bell: ignore stylistic issues if it's not integral to the language
+
+#### Examples
+
+- **Ruby** has a language feature where `_` is treated differently,
+  - :-1: If a student uses `_` for a variable name, but then uses it.
+- **Ruby** reconises `constants` only if they start with a **C**apital Letter,
+  - :-1: If a student uses `snake_case` for a `class` name
+- **Ruby** has _defacto_ standards on `cAsInG` and `name-ing`,
+  - :no_bell: If a student uses `camelCase` for a variable, leave it be.
+  - :speech_balloon: You may guide students that `snake_case` is to be expected by
+    most IDE's and highlighting on exercism in code blocks for variables.
```

Comment:
I believe this is meant to be plural and not possessive.  ~IDE's~ IDEs

Reply:
Yes it is!